app.rb

```diff
@@ -0,0 +1,112 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'classroom'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_reader :books, :people, :rentals, :classroom
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @classroom = Classroom.new('college')
+  end
+
+  # Method to list all books
+  def list_all_books()
+    puts 'All Books:'
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  # Method to list all people
+  def list_all_people()
+    puts 'All People:'
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
+
+  # Method to create a person
+  def create_person()
+    puts 'Enter person name:'
+    name = gets.chomp
+    puts 'Enter person age:'
+    age = gets.chomp.to_i
+    puts 'Is it a student? (y/n):'
+    is_student = gets.chomp.downcase == 'y'
+
+    if is_student
+      puts 'Enter student classroom:'
+      classroom = gets.chomp
+      person = Student.new(age, name, classroom, parent_permission: true)
+    else
+      puts 'Enter teacher specialization:'
+      gets.chomp
+      person = Person.new(age, name: name, parent_permission: true)
+    end
+
+    people << person
+    puts 'Person created successfully.'
+  end
```

Comment:
- [ ] Good job so far 👍👍 However, as mentioned by the previous reviewer [here](https://github.com/Naledi-Dikgale/oop-school-library/pull/4#discussion_r1237108849) and according to the project requirements, we need to **_Create a person (teacher or student, not a plain Person)._** ✨ But in the current implementation, users aren't given the option to create a teacher or a student, but asked `Is it a student? (y/n):` Please note that this is not in accordance with the given [referenced video](https://www.youtube.com/watch?v=vkkgrhD6aXQ), kindly fix this implementation 🤓⛏

Current UI | Expected UI
---| ---|
![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/106119105/0c7eb5c5-5cbd-4d60-930e-cf90cda3a168) | ![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/106119105/06744ead-5891-4f8a-9f1f-9a98c51d6cef)

- [ ] Also, as suggested earlier, the person's details are not being displayed as seen in the video and it is hard to differentiate between students and teachers  so kindly fix that as well ✨📌

Current UI | Expected UI
---| ---|
![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/106119105/5da1f0e1-c0ac-4d6e-b17e-f05edbf4d973) | ![image](https://github.com/Naledi-Dikgale/oop-school-library/assets/106119105/79333893-8a80-4fee-a964-b1e8484f27d1)